### PR TITLE
deps: update google-cloud-bigtable dependencies

### PR DESCRIPTION
**Pull request by mutianf**
Bridge #3912 and disabling channel priming by default.

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/SharedDataClientWrapperFactory.java [hunk 1]
```diff
@@ -76,6 +72,7 @@ synchronized DataClientWrapper createDataClient(BigtableHBaseVeneerSettings sett
       Builder builder = settings.getDataSettings().toBuilder();
       builder
           .stubSettings()
+          .setRefreshingChannel(false)
```
Comment (igorbernstein2) [inline bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/SharedDataClientWrapperFactory.java:75]: TODO

**Review by igorbernstein2 — APPROVED** [review]
lgtm, but please add a todo to re-enable the priming

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml [hunk 2]
```diff
@@ -36,7 +36,7 @@ limitations under the License.
 
   <properties>
     <mrunit.version>1.1.0</mrunit.version>
-    <google-cloud-bigtable-emulator.version>0.154.1</google-cloud-bigtable-emulator.version>
+    <google-cloud-bigtable-emulator.version>0.155.1</google-cloud-bigtable-emulator.version>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml:39]: please update version here

**Review by kolea2 — APPROVED** [review]
lgtm after last comment

**Pull request merged**